packages/fxa-content-server/tests/functional/settings_v2/navigation.js

```diff
@@ -0,0 +1,125 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+'use strict';
+
+const intern = require('intern').default;
+const { describe, it, beforeEach } = intern.getPlugin('interface.bdd');
+const selectors = require('../lib/selectors');
+const FunctionalHelpers = require('../lib/helpers');
+
+const config = intern._config;
+const EMAIL_FIRST = config.fxaContentRoot;
+const SETTINGS_V2_URL = `${config.fxaContentRoot}beta/settings`;
+const password = 'passwordzxcv';
+
+const { createEmail } = FunctionalHelpers;
+
+const {
+  clearBrowserState,
+  click,
+  createUser,
+  openPage,
+  fillOutEmailFirstSignIn,
+  testElementExists,
+} = FunctionalHelpers.helpersRemoteWrapped;
+
+describe('navigation', () => {
+  let email;
+  beforeEach(async ({ remote }) => {
```

Comment:
I think for simpler test suites like nav we should only do the signup/in part once in a `before` and let the `beforeEach` do a fresh navigation to the settings page. It should be noticeably faster that way.